spec/chapters/controllers.tex

```diff
@@ -329,3 +329,17 @@ \section{Views}
 depending on which controller from Section \ref{models} triggered this view's 
 processing.
 
+\subsection{Building URIs in a View}
+\label{mvc_uri}
+
+In {\tt views} links and form actions require a URI. To avoid repeating the declarative
```

Comment:
Perhaps remove the `\tt` here? I see no reason why the word should be monospace. Or am I missing something here?